packages/server/billing/helpers/terminateSubscription.ts

```diff
@@ -18,7 +18,7 @@ const terminateSubscription = async (orgId: string) => {
             stripeSubscriptionId: null
           },
           {returnChanges: true}
-        )('changes')(0)('old_val')
+        )('changes')(0)('old_val')('stripeSubscriptionId')
```

Comment:
-1 this seems to contradict how we're reading the `rethinkResult` in line l26

Reply:
I wasn't sure what you meant by contradiction, but I made a few changes to make it more clear & easy to read. Let me know if that's what you were thinking?

Reply:
Thanks for the change, now I understand what you were doing.